Makefile_rke2

```diff
@@ -49,7 +49,7 @@ attack: copy-scripts create-bad exec
 redpanda: helm 
 	-$(HELM) repo add redpanda https://charts.redpanda.com
 	-$(HELM) repo update
-	-$(HELM) upgrade --install redpanda-src redpanda/redpanda -n redpanda --create-namespace --values redpanda/rke2values.yaml --set image.repository=docker.redpanda.com/redpandadata/redpanda-unstable --set image.tag=v24.1.1-rc4
+	-$(HELM) upgrade --install redpanda-src  redpanda-data/redpanda --version 5.8.8 -n redpanda --create-namespace --values redpanda/diffvalues.yaml 
```

Comment:
Is `redpanda/rke2values.yaml` also not needed anymore? Then we could delete it as well.